pages/common/netcat.md

```diff
@@ -1,6 +1,7 @@
 # netcat
 
 > This command is an alias of `nc`.
+> More information: <https://manned.org/man/nc.1>.
```

Comment:
```suggestion
> More information: <https://manned.org/nc>.
```

Reply:
This is the English one

Reply:
The suggested links redirects to the original one.
https://manned.org/nc ➡️ https://manned.org/man/nc.1
Also, that would mean changing the link on all other languages.

Reply:
Some time ago we changed most of the links from manned.org/man/* to manned.org/* so we should update this too

Reply:
Ok, I'll update the link for this PR then do another PR to change all links.